### DISCUSSION TITLE
App • ⚠️  Extension(@quasar/qzoom): is not compatible with quasar v2.7.5. Required version: ^1.5.0

### DISCUSSION THREAD

**Discussion by meo1** [quasarframework/quasar#13976] [Quasar App Extensions] [answered]
I have started getting this message:  

App • ⚠️  Extension(@quasar/qzoom): is not compatible with quasar v2.7.5. Required version: ^1.5.0

![image](https://user-images.githubusercontent.com/35503088/179391948-b55f7e0f-a662-4644-866a-83538a2bf7d0.png)

**Comment by MilosPaunovic** [comment, marked as answer]
That extension is not ported to `Quasar@2.x.x`.

**Reply by meo1** [reply]
Will it be ported in the near future?

**Reply by MilosPaunovic** [reply]
I wouldn't count on it.
